source/views/contacts/contact-detail.js

```diff
@@ -41,14 +46,17 @@ export class ContactsDetailView extends React.PureComponent {
     }
   }
 
-  navigation: {state: {params: {contact: ContactType}}}
+  props: {navigation: {state: {params: {contact: ContactType}}}}
 
   onPress = () => {
     const item = this.props.navigation.state.params.contact
-    tracker.trackScreenView(
-      `ImportantContacts_${item.title.replace(' ', '')}View`,
-    )
-    promptCall(item.buttonText, item.phoneNumber)
+    const {title, phoneNumber, buttonText, buttonLink} = item
```

Comment:
Just combine this line with the previous one, please.

Reply:
Done.